var/spack/repos/builtin/packages/pythia6/package.py

```diff
@@ -33,11 +33,13 @@ class Pythia6(CMakePackage):
     """
 
     homepage = 'https://pythiasix.hepforge.org/'
-    url = 'http://www.hepforge.org/archive/pythiasix/pythia-6.4.28.tgz'
+    url = 'https://pythia.org/download/pythia6/pythia6428-split.tgz'
 
     tags = ['hep']
 
-    version('6.4.28',
+    maintainers = ['gartung', 'chissg']
+
+    version('6.4.28', url='https://pythia.org/download/pythia6/pythia6428-split.tgz',
```

Comment:
Is it possible to remove `url` here, since it's the same as the one at class level?

Reply:
We'll need a url_for_version since the version doesn't use periods.

Reply:
Well spotted